djorg/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for djorg project.
+
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '5!b1zy)c_j%pvj#l8kqtd9*59bu(+*p2zd4dr4dq+t38=#@z+3'
```

Comment:
you don't want to share your secret key exposed like this, look into .env on how to protect your secret key.